packages/terra-pills/.npmignore

```diff
@@ -0,0 +1,9 @@
+*.log
```

Comment:
Instead of using a .npmignore lets use the files field in package.json like this: https://github.com/cerner/terra-core/blob/main/packages/terra-core-docs/package.json#L28-L30